### PR TITLE
Move supportFeature to mcdata

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by u9g** [comment]
requires https://github.com/PrismarineJS/minecraft-data/pull/521

**Comment by rom1504** [comment]
can you add doc ?

**Comment by u9g** [comment]
> can you add doc ?

done

File: doc/api.md [hunk 1]
```diff
@@ -337,6 +337,10 @@ Mapping from 1.12 block:metadata to 1.13 block names
 
 examples 0:0 -> minecraft:air
 
+### minecraft-data.supportFeature()
```
Comment (rom1504) [inline doc/api.md:340]: add input/output

Comment (u9g) [reply doc/api.md]: done

**Comment by rom1504** [comment]
why are the tests not failing?

File: index.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ module.exports = function (mcVersion, preNetty) {
   nmcData.isOlderThan = version => nmcData.version['<'](version)
   nmcData.version = Object.assign(majorVersion, nmcData.version)
   cache[majorVersion.type + '_' + majorVersion.majorVersion] = nmcData
-
+  nmcData.supportFeature = feature => require('./lib/supportsFeature')(feature, nmcData.version.minecraftVersion)
```
Comment (rom1504) [inline index.js:66]: let's do require('./lib/supportsFeature') at the top level

Comment (u9g) [reply index.js]: done :)

**Pull request merged**